libretro_core_options.h

```diff
@@ -1125,11 +1125,11 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(core_timing_fps),
       "Core-Reported FPS Timing",
       NULL,
-      "Sets FPS timing that the core will report to the frontend. Automatic toggling will allow the core to switch between reporting progressive and interlaced rates, but may cause frontend video/audio driver reinits.",
+      "Choose the FPS timing that the core will report to the frontend. Automatic Toggling will allow the core to switch between reporting progressive and interlaced rates, but may cause frontend video/audio driver re-inits.",
```

Comment:
"re-inits" might be confusing, perhaps?

Reply:
I'm not 100% sure, but maybe this'll work:
"[...] but may cause frontend video/audio drivers to restart."